packages/blocks/src/api/serializer.js

```diff
@@ -95,27 +103,42 @@ export function getSaveElement(
 	}
 
 	let element = save( { attributes, innerBlocks } );
+	let blockPropsFilter = ( props ) => props;
+
+	if ( hasFilter( 'blocks.getSaveContent.extraProps' ) ) {
+		if ( hasBlockSupport( blockType, 'lightBlockWrapper', false ) ) {
+			blockPropsFilter = ( props ) =>
+				/**
+				 * Filters the props applied to the block save result element.
+				 *
+				 * @param {Object}  props      Props applied to save element.
+				 * @param {WPBlock} blockType  Block type definition.
+				 * @param {Object}  attributes Block attributes.
+				 */
+				applyFilters(
+					'blocks.getSaveContent.extraProps',
```

Comment:
One thing I was wondering is whether we should just apply this hook to the Edit version of `Block.*`

Reply:
We could, but we'd need more context and then the hook namespace also doesn't make sense anymore.